REST_based_Auth_API/my_rest_api/core/views.py

```diff
@@ -0,0 +1,371 @@
+from decimal import Context
+import json
+
+from django.db.models.expressions import OrderBy
+from django.http import request
+from django.shortcuts import render
+
+from rest_framework import generics, status, viewsets
+from rest_framework.decorators import action
+from rest_framework import generics
+from rest_framework.fields import NullBooleanField
```

Comment:
Faraz, the purpose of this whole assignment is to see how diligent you are at following best code practices. I have asked you for the umteenth time to remove unused imports. Be mindful of small things like this because they all contribute to your assessment.